torch/nn/modules/loss.py

```diff
@@ -2062,10 +2062,11 @@ class CTCLoss(_Loss):
         https://www.cs.toronto.edu/~graves/icml_2006.pdf
 
     Note:
-        In order to use CuDNN, the following must be satisfied: :attr:`targets` must be
+        In order to use CuDNN, the following must be satisfied: the :attr:`targets` must be
         in concatenated format, all :attr:`input_lengths` must be `T`.  :math:`blank=0`,
         :attr:`target_lengths` :math:`\leq 256`, the integer arguments must be of
-        dtype :attr:`torch.int32`.
+        dtype :attr:`torch.int32`, and the :attr:`input` itself must be of
```

Comment:
To match the name used above under the Shape: section

```suggestion
        dtype :attr:`torch.int32`, and the :attr:`log_probs` itself must be of
```